ee/benchmarks/README.md

```diff
@@ -42,6 +42,14 @@ asv machine --machine ci-benchmarks --config ee/benchmarks/asv.conf.json
 CLICKHOUSE_HOST=X CLICKHOUSE_USER=X CLICKHOUSE_PASSWORD=X CLICKHOUSE_DATABASE=posthog asv run --config ee/benchmarks/asv.conf.json
 ```
 
+You'll probably want to be running one test, with quick iteration. Running e.g.:
+
+```
+asv run --config ee/benchmarks/asv.conf.json --bench track_lifecycle --quick
```

Comment:
I've also used --environment existing to skip the virtualenv setup

Reply:
Ah great I missed that one 😳